### DISCUSSION TITLE
Can I Remove schema qualification from the data editor query?

### DISCUSSION THREAD

**Discussion by wowsonsun** [dbeaver/dbeaver#12394] [Q&A] [answered]
#### System information: 
- deepin 20.1
- DBeaver enterprise  7.3.0
- Additional extensions

#### Connection specification:
- MySQL 5.6.0-UDAL-DBPROXY-2.4.0-2018-09-06T12:22:21Z
- MySQL Connector Java mysql-connector-java-5.1.48 ( Revision: 29734982609c32d3ab7e5cac2e6acee69ff6b4aa )
- 

#### Describe the problem you're observing:

the MySQL middleware we're using does not support scheme qualifier, either.
In addition, the UPDATE/DELETE clause generated when submitting changes made in Data Views should not include scheme qualifier for the same reason.

![image](https://user-images.githubusercontent.com/23351005/110412625-7550aa00-80c7-11eb-8884-f6800dfdf691.png)
![image](https://user-images.githubusercontent.com/23351005/110412907-ee500180-80c7-11eb-8f1f-a5231b40777b.png)

**Comment by kseniiaguzeeva** [comment, marked as answer]
We already have such feature.
Press **Script** on the bottom toolbar after editing a table (before saving changes). Separate window will be opened with generated SQL expression. You can remove schema name there and copy the expression.
![Screenshot 2021-03-25 at 10 37 45](https://user-images.githubusercontent.com/51405061/112435790-42104b00-8d56-11eb-9bea-d5d1d191facf.png)

Please ask to reopen the ticket if my advice doesn't cover your needs

**Comment by wowsonsun** [comment]
**use fully qualified names** is disabled
![image](https://user-images.githubusercontent.com/23351005/116359639-9d3fcc80-a831-11eb-991e-c5e2192bd38b.png)

**Comment by wowsonsun** [comment]
@kseniiaguzeeva

**Comment by LonwoLonwo** [comment]
Hello @wowsonsun 
What DBeaver version do you use?
We fixed this problem several releases back.

![2021-05-06 11_14_55-DBeaver Enterprise 21 0 4 - customer](https://user-images.githubusercontent.com/45152336/117264872-56b72700-ae5c-11eb-9b52-16db0f5e9e54.png)

**Comment by wowsonsun** [comment]
DBeaver Enterprise 21.0.0 @LonwoLonwo

**Comment by LonwoLonwo** [comment]
@wowsonsun can you test [Early access](https://dbeaver.com/files/ea/enterprise/) version, please?

**Comment by wowsonsun** [comment]
perfect! The problem is solved
![image](https://user-images.githubusercontent.com/23351005/117597895-1ff15180-b179-11eb-850a-481ea4ece7d6.png)
@LonwoLonwo
